src/index.ts

```diff
@@ -31,6 +31,14 @@ type MakeUniversalOpts = {
    * Forcefully overwrite any existing files that are in the way of generating the universal application
    */
   force: boolean;
+  /**
+   * Fuse x64 and arm64 ASARs into one.
+   */
+  fuseASARs?: boolean;
+  /**
+   * Minimatch pattern of paths that are allowed to be unique in ASARs.
+   */
+  fuseASARsAllowList?: string;
```

Comment:
What happens if they're on this list, does it take the x64 file? the arm64 file? etc.

I think it would be better to somehow allow folks to choose, or preferably not allow this at all.  It leads to uncertainty, you should just form your ASAR file correctly.

Reply:
Ah I see, so these files to be clear exist in one asar but do not exist in the other? 🤔 I don't know what a better name is but it's definitely not clear what that does if I didn't grok it first try.

Maybe `singleArchFiles` or something idk, open to ideas here.

Reply:
> Ah I see, so these files to be clear exist in one asar but do not exist in the other? 🤔 I don't know what a better name is but it's definitely not clear what that does if I didn't grok it first try.

You got it right.

> Maybe `singleArchFiles` or something idk, open to ideas here.

`singleArchFiles` certainly sounds better than what I came up with, let me change it to this. I'm out of ideas, otherwise!

Reply:
Renamed.